### PR TITLE
 moved away common.grade

### PR DESCRIPTION
**Pull request by dsnyder-fairway**

### CODE REVIEW

File: webapp/src/main/webapp/package.json [hunk 1]
```diff
@@ -28,7 +28,6 @@
     "@ngx-translate/http-loader": "^2.0.0",
     "@sbac/sbac-ui-kit": "0.0.24",
     "@swimlane/ngx-dnd": "3.1.3",
-    "angular-2-dropdown-multiselect": "^1.6.3",
```
Comment (dsnyder-fairway) [inline webapp/src/main/webapp/package.json]: removed this unused dependency

File: webapp/src/main/webapp/src/app/student/results/tables/student-history-ica-summative-table.component.html [hunk 2]
```diff
@@ -40,7 +40,7 @@
   <p-column field="exam.enrolledGrade" header="{{'labels.student.results.cols.enrolled-grade' | translate}}"
             sortable="true">
     <ng-template let-exam="rowData.exam" pTemplate="body">
-      {{ 'common.grade.' + exam.enrolledGrade + '.short-name' }}
+      {{ 'common.assessment-grade-short-label.' + exam.enrolledGrade | translate }}
```
Comment (dsnyder-fairway) [inline webapp/src/main/webapp/src/app/student/results/tables/student-history-ica-summative-table.component.html]: noticed I forgot to translate something

Comment (dsnyder-fairway) [reply webapp/src/main/webapp/src/app/student/results/tables/student-history-ica-summative-table.component.html]: should have the same prefix as `student-history-iab-table.component.html`, no? `common.enrolled-grade-name.`? Not sure why this drifted apart

Comment (esotrope) [reply webapp/src/main/webapp/src/app/student/results/tables/student-history-ica-summative-table.component.html]: looks like a bug here - i think we are showing an enrolled grade as an assessment grade which is wrong
probably need to show it as an enrolled grade

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 3]
```diff
@@ -409,127 +409,93 @@
...
+      "13": "13",
+      "IT": "IT",
+      "KG": "KG",
+      "PK": "PK",
+      "PR": "PR",
+      "PS": "PS",
+      "TK": "TK",
+      "UG": "UG"
+    },
+    "enrolled-grade-name": {
```
Comment (esotrope) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: nit: name-> label

**Pull request merged**